_rules/html-page-lang-matches-default-ucwvc8.md

```diff
@@ -0,0 +1,278 @@
+---
+id: ucwvc8
+name: HTML page language subtag matches default language
+rule_type: atomic
+description: |
+  This rule checks that the primary language subtag of the page language matches the default language of the page
+accessibility_requirements:
+  wcag20:3.1.1: # Language of Page (A)
+    forConformance: true
+    failed: not satisfied
+    passed: satisfied
+    inapplicable: further testing needed
+  wcag-technique:H57: # Using the language attribute on the HTML element
+    forConformance: false
+    failed: not satisfied
+    passed: satisfied
+    inapplicable: satisfied
+input_aspects:
+  - DOM tree
+  - Accessibility tree
+  - CSS Styling
+  - Language
+acknowledgments:
+  authors:
+    - Wilco Fiers
+---
+
+## Applicability
+
+This rule applies to any [document element][] if it is an `html` element for which all of the following are true:
+
+- The [document element][] has a `lang` attribute with a value that is a [valid language tag][]; and
+- The [document element][] is in a [top-level browsing context][]; and
+- The [document element][] has a [content type][] of `text/html`; and
+- The [document element][] has a defined [default page language][].
+
+## Expectation
+
+For each test target, the [primary language][] of the [valid language tag][] matches the [default page language][] of the test target.
```

Comment:
This is intentional. I think this reads better. Why do you think this needs to change?

Reply:
The definition we are referring to here redirects us to the "primary language subtag" and on the other hand, primary language subtag refers to the language not the complete tag with the country. For example, primary language can be `en-GB` and `en-US` but the primary subtag would be `en`, not `GB` as it's a region where it used which specifies the language itself.
I guess I misunderstood that term here, you meant the primary language as general whether simple language subtag or language with other subtags such as `en` or `en-US` respectively.
Please correct me if I'm still misunderstanding it.

Reply:
It's a specifier. A language tag only consists of subtags, so adding it doesn't provide additional meaning. It's like saying "The child node of the node", you might as well say "the child of the node", because all children of a DOM node are themselves nodes. Both phrases mean the same thing, but the first one has unnecessary specificity in it.

If you insist, I'll take on the change, but I don't think it's necessary.

Reply:
Well explained. No need to change it. 👍